src/afl-forkserver.c

```diff
@@ -238,6 +238,7 @@ void afl_fsrv_init(afl_forkserver_t *fsrv) {
   fsrv->mem_limit = MEM_LIMIT;
   fsrv->out_file = NULL;
   fsrv->child_kill_signal = SIGKILL;
+  fsrv->custom_fuzz_run = NULL;
```

Comment:
you are not setting the shm_id here, nor copying it over in the init_dup() function

Reply:
Oh! I missed it.
Fixed. Thanks!